requirements-dev.txt

```diff
@@ -1,15 +0,0 @@
-black
```

Comment:
please do nor remove or change this file!

Reply:
Added the file back